### PR TITLE
Refactor: Polish 2.2.x governance doc

### PR DESCRIPTION
**Pull request by 123liuziming**
Polish 2.2.x governance document after pilot-agent is supported.

### CODE REVIEW

**Review by ruansheng8 — APPROVED** [review]

File: spring-cloud-alibaba-docs/src/main/asciidoc-zh/governance.adoc [hunk 1]
```diff
@@ -9,17 +9,51 @@ image::pic/resource-transform.png[]
...
+如果在项目中使用 `Istio` 来实现规则转换，首先注意需要搭建一个Kubernetes集群，并且在其中部署Istio，具体参考 https://istio.io/latest/zh/docs/setup/install[Istio安装]。然后在需要接收到来自Istio规则的应用中添加如下starter依赖：
 [source,xml,indent=0]
 ----
 <dependency>
   <groupId>com.alibaba.cloud</groupId>
   <artifactId>spring-cloud-starter-xds-adapter</artifactId>
 </dependency>
 ----
-OpenSergo请参考 https://github.com/alibaba/spring-cloud-alibaba/tree/2.2.x/spring-cloud-alibaba-examples/governance-example/label-routing-example[Spring Cloud Alibaba Routing Examples]
+OpenSergo请参考 https://github.com/alibaba/spring-cloud-alibaba/tree/2.2.x/spring-cloud-alibaba-examples/governance-example/label-routing-example[Spring Cloud Alibaba Routing Examples]。
```
Comment (steverao) [inline spring-cloud-alibaba-docs/src/main/asciidoc-zh/governance.adoc:20]: There is a Chinese signal in your English sentence.

File: spring-cloud-alibaba-docs/src/main/asciidoc/governance.adoc [hunk 2]
```diff
@@ -10,17 +10,51 @@ image::pic/resource-transform.png[]
...
   <artifactId>spring-cloud-starter-xds-adapter</artifactId>
 </dependency>
 ----
-If you want to use OpenSergo, you can refer to https://github.com/alibaba/spring-cloud-alibaba/tree/2.2.x/spring-cloud-alibaba-examples/governance-example/label-routing-example[Spring Cloud Alibaba Routing Examples]
+If you want to use OpenSergo, you can refer to https://github.com/alibaba/spring-cloud-alibaba/tree/2.2.x/spring-cloud-alibaba-examples/governance-example/label-routing-example[Spring Cloud Alibaba Routing Examples].
 
-After that, configure the following configuration in the application.yml:
+There are two ways to connect to `Istio` control plane:
+
+==== Inject pilot-agent(recommend)
```
Comment (steverao) [inline spring-cloud-alibaba-docs/src/main/asciidoc/governance.adoc]: Recommended?

File: spring-cloud-alibaba-examples/governance-example/label-routing-example/readme.md [hunk 3]
```diff
@@ -178,42 +178,16 @@ Please refer to [install](https://istio.io/latest/zh/docs/setup/install/) chapte
...
-        port: ${ISTIOD_PORT:15010}
-        # Istiod thread-pool size
-        polling-pool-size: ${POLLING_POOL_SIZE:10}
-        # Istiod polling gap
-        polling-time: ${POLLING_TIME:10}
-        # Istiod token(For Istio 15012 port)
-        istiod-token: ${ISTIOD_TOKEN:}
-        # Whether to print xds log
-        log-xds: ${LOG_XDS:true}
+        enabled: ${ISTIO_AUTH_ENABLE:true}
```
Comment (steverao) [inline spring-cloud-alibaba-examples/governance-example/label-routing-example/readme.md]: This line isn't consistent with Chinese version.

**Comment by 123liuziming** [comment]
I will fix them tomorrow.

File: spring-cloud-alibaba-docs/src/main/asciidoc-zh/governance.adoc (hunk 1 above)
Comment (123liuziming) [reply spring-cloud-alibaba-docs/src/main/asciidoc-zh/governance.adoc:20]: It is in asciidoc-zh directory.

**Comment by yuluo-yx** [comment]
documentation has been migrated to the documentation website, so closed

**Pull request closed without merging**